org.sil.Bloom.yml

```diff
@@ -0,0 +1,563 @@
+                  sha256: 32f1eba1c2df67bc55920cd45ce3689c32a5c705ae01fd7ead52ca180221b469
+            buildsystem: simple
+            build-commands:
+                - ar x msbuild_*.deb
+                - tar xf data.tar.xz
+                - sed -e 's|/usr/|/usr/lib/sdk/mono6/|g' -i usr/bin/msbuild
+                - cp -a usr/. /usr/lib/sdk/mono6/
+          - libpalaso.yml
+          - name: ca-certificates
+            sources:
+                - type: file
+                  url: http://us.archive.ubuntu.com/ubuntu/pool/main/c/ca-certificates/ca-certificates_20211016~20.04.1_all.deb
+                  sha256: 5a7b922d35e328d7d7765a353f1bbc9e14c8476e67606b605c6d0b1186037571
+            buildsystem: simple
+            build-commands:
+                - |
+                    set -xueo pipefail
+                    ar x ca-certificates_*all.deb
+                    tar xf data.tar.xz
+                    cat usr/share/ca-certificates/mozilla/*.crt >ca-certificates.crt
+                    install -d /app/share/ca-certificates
+                    install ca-certificates.crt /app/share/ca-certificates
+          - name: geckofx
+            buildsystem: simple
+            build-commands:
+                - |
+                    set -xueo pipefail
+                    pwd
+                    ls -FC
+                    msbuild /p:Configuration=Release_Linux /p:Platform="Any CPU" Geckofx.sln
+                    # copy where these can be found by the BloomDesktop nuget package code
+                    install -d /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Core.dll /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Core.dll.config /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Core.pdb /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Winforms.dll /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Winforms.dll.config /app/geckofx
+                    install Geckofx-Winforms/bin/Release_Linux/Geckofx-Winforms.pdb /app/geckofx
+            sources:
+                - type: git
+                  url: https://github.com/StephenMcConnel/Geckofx60.git
+                  commit: a77e5ba72728630ee02b95892d5fd0c219f87c98
+                - type: patch
+                  path: patches/geckofx/FixesForFlatpak.patch
+    - name: ArtOfReading
+      sources:
+          # This .deb contains only PNG image files with a few text files such
+          # as an index and changelog.
+          - type: file
+            url: https://packages.sil.org/ubuntu/pool/main/a/art-of-reading3/art-of-reading3_3.3.1+bionic1_all.deb
+            sha256: 96d99d60c1838aaac8461e3568185cc287206d2852bd3e11d94eef5d9d6a3f15
+      buildsystem: simple
+      only-arches:
+          - x86_64
```

Comment:
```suggestion
```

Reply:
removed in both places.  added a flathub.json file